ramls/inventory-hierarchy/inventory-items-and-holdings.json

```diff
@@ -0,0 +1,804 @@
+                  "type": "string"
+                }
+              },
+              "required": [
+                "uri",
+                "name",
+                "linkText",
+                "publicNote",
+                "relationshipId",
+                "materialsSpecification"
+              ]
+            }
+          },
+          "notes": {
+            "description": "Notes about action, copy, binding etc.",
+            "type": "array",
+            "items": {
+              "type": "object",
+              "properties": {
+                "itemNoteTypeName": {
+                  "description": "Name of the item note type",
+                  "type": "string"
+                },
+                "note": {
+                  "description": "Text content of the note",
+                  "type": "string"
+                }
+              },
+              "required": [
+                "itemNoteTypeName",
+                "note"
+              ]
+            }
+          },
+          "tags": {
+            "description": "Arbitrary tags associated with this item",
+            "type": "object",
+            "properties": {
+              "tagList": {
+                "description": "List of tags",
+                "type": "array",
+                "items": {
+                  "important": {
+                    "description": "Indicate if tag is important",
+                    "type": "string"
+                  }
+                }
+              }
+            }
+          },
+          "statisticalCodes": {
+            "description": "Item record statistical codes",
+            "type": "array",
+            "items": {
+              "type": "object",
+              "properties": {
+                "id": {
+                  "description": "Unique ID of the statistical code",
+                  "type": "string",
+                  "$ref": "../uuid.json"
+                },
+                "code": {
+                  "description": "Statistical code, a distinct label",
+                  "type": "string"
+                },
+                "name": {
+                  "description": "Name or description of a statistical code",
+                  "type": "string"
+                },
+                "statisticalCodeType": {
+                  "description": "Name of a statistical code type",
+                  "type": "string"
+                },
+                "source": {
+                  "description": "Label indicating where the statistical code type entry originates from, i.e. 'folio' or 'local'",
+                  "type": "string"
+                }
+              },
+              "required": [
+                "id",
+                "code",
+                "name",
+                "statisticalCodeType",
+                "source"
+              ]
+            }
+          }
+        },
+        "required": [
```

Comment:
Have these schema be generated in some way? The required properties defined here don't seem to match up to the properties that are required in the underlying records and so I don't understand how these expectations can be fulfilled.

Reply:
I'll check requirted fields from mod-inventory-storage's json schema definitions anf fix this. Thank you.